build/check-style.sh

```diff
@@ -95,7 +95,7 @@ TestImportNames() {
 }
 
 TestIneffassign() {
-  ! ineffassign . | grep -vF 'ineffectual assignment to sqlDollar' | grep -vF '.pb.go' # https://github.com/gogo/protobuf/issues/149
+  ! (ineffassign . || true) | grep -vF 'ineffectual assignment to sqlDollar' | grep -vF '.pb.go' # https://github.com/gogo/protobuf/issues/149
```

Comment:
wait, doesn't this mean that all of these will immediately fail under pipefail? since we're returning 1 from the first part of the pipe.